spec/models/mediaflux/version_request_spec.rb

```diff
@@ -5,13 +5,15 @@
   subject(:request) { described_class.new(session_token: session_token) }
   let(:session_token) { user.mediaflux_session }
   let(:user) { FactoryBot.create(:user) }
+  let(:docker_response) { "4.16.032" }
```

Comment:
No change requested, just noting this is sad that we have two different versions here.